java/feathub-udf/flink-udf/src/main/java/com/alibaba/feathub/flink/udf/processfunction/SlidingWindowKeyedProcessFunction.java

```diff
@@ -238,7 +283,7 @@ private static class SlidingWindowState {
         private final AggregationFieldsDescriptor aggregationFieldsDescriptor;
 
         /** This MapState maps from row timestamp to the row. */
-        private final MapState<Long, Row> timestampToRow;
+        private final MapState<Long, List<Row>> timestampToRow;
```

Comment:
Could you also rename the variable to `timestampToRows` and also the docs to reflect the change?